spec/javascripts/packages/one-time-code-input/index-spec.js

```diff
@@ -104,4 +104,32 @@ describe('OneTimeCodeInput', () => {
       expect(navigator.credentials.get).not.to.have.been.called();
     });
   });
+
+  context('Otp Hidden input created', () => {
```

Comment:
Nit: Same effect, but I might suggest using `describe` here instead of `context`, as a way to communicate that this is testing a common behavior, not one which is conditional based on some contextual setup.

```suggestion
  describe('Otp Hidden input created', () => {
```